pandas/io/parsers/readers.py

```diff
@@ -1256,8 +1173,6 @@ def read_table(
     encoding_errors: str | None = "strict",
     dialect: str | csv.Dialect | None = None,
     # Error Handling
-    error_bad_lines: bool | None = None,
-    warn_bad_lines: bool | None = None,
     # TODO(2.0): set on_bad_lines to "error".
     # See _refine_defaults_read comment for why we do this.
     on_bad_lines=None,
```

Comment:
Can you change on_bad_lines to "error" here and in read_csv?

Reply:
good catch, thx, updated both